PYME/experimental/triangle_mesh_utils.c

```diff
@@ -58,7 +58,7 @@ DB: Performance suggestions:
 
 */
 
-static void update_single_vertex_neighbours(int v_idx, halfedge_t *halfedges, void *vertices_, void *faces_)
+static void update_single_vertex_neighbours(int32_t v_idx, halfedge_t *halfedges, void *vertices_, void *faces_)
```

Comment:
`v_idx` should be `int` (used here as an index to an array)

Reply:
Fair enough.